src/openst.js

```diff
@@ -68,9 +70,53 @@ class OpenST {
       createAndAddModules: createAndAddModules.address,
       tokenRules: tokenRulesAddress,
     };
+
     Object.assign(this.chainConfig.openst, setupData);
+
+    this.chainConfig.utilityBrandedTokens[ubtConfig.index].openst = setupData;
+
     logger.info('Completed Setup of OpenST');
   }
+
+  getUtilityBrandedTokenConfig(auxiliaryEIP20Token) {
+    for (let i = 0; i < this.chainConfig.utilityBrandedTokens.length; i++) {
+      const ut = this.chainConfig.utilityBrandedTokens[i];
+      if (ut.address === auxiliaryEIP20Token) {
+        return { ubt: ut, index: i };
+      }
+    }
+    return 0;
+  }
+
+  async deployPricerRule(
+    auxiliaryEIP20Token,
+    baseCurrencyCode,
+    conversionRate,
+    conversionRateDecimals,
+    requiredPriceOracleDecimals,
+  ) {
+    logger.info('Deployment of PricerRule');
+    const ubtConfig = this.getUtilityBrandedTokenConfig(auxiliaryEIP20Token);
+
+    const pricerRule = await PricerRule.deploytest(
+      this.auxiliary.web3,
+      ubtConfig.ubt.organizationAddress,
+      ubtConfig.ubt.address,
+      ubtConfig.ubt.openst.tokenRules,
+      baseCurrencyCode,
+      conversionRate,
+      conversionRateDecimals,
+      requiredPriceOracleDecimals,
+      this.auxiliary.txOptions,
+    );
+
+    const pricerOracleData = {
+      address: pricerRule.address,
+    };
+
+    this.chainConfig.utilityBrandedTokens[ubtConfig.index].pricerOracle = pricerOracleData;
```

Comment:
Here we are writing PriceOracles at economy level. 
There can be multiple PriceOracles deployed in an economy. We should update writing to config Accordingly. Above code can look like below:
```
this.chainConfig.utilityBrandedTokens[ubtConfig.index].pricerOracles = pricerOracleData;
```